packages/renderer/src/lib/configmaps-secrets/ConfigMapSecretUI.ts

```diff
@@ -16,7 +16,9 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-export interface ConfigMapSecretUI {
+import type { KubernetesUI } from '../objects/KubernetesUI';
+
+export interface ConfigMapSecretUI extends KubernetesUI {
   name: string;
   namespace: string;
```

Comment:
I suppose you can remove name and namespace here (and for the other resources)

Reply:
As per the comment below I removed `name` but did not remove `namespace`, because it is optional on `KubernetesObjectUI` but is required for all sub-interfaces except `NodeUI`.